src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator
+
+import calculator.type.Operator
+import calculator.validator.OperationValidator
+
+class Calculator {
+    fun calculate(expression: String): Int {
+        val expressionList = extractExpressionList(expression)
+
+        var result = expressionList[0].toInt()
+        for (index in 1 until expressionList.size step 2) {
+            val operator = Operator.of(expressionList[index])
+            if (operator != Operator.NONE) {
+                val rightOperand = expressionList.getOrNull(index + 1)?.toIntOrNull() ?: continue
+                result = performOperation(result, rightOperand, operator)
```

Comment:
어짜피 한번만 호출하면되는 경우라면 굳이 메서드로 쪼갤 필요 없을 거 같아요. 🤔

```suggestion
                result = operator.operationFunction(result, rightOperand)
```

Reply:
그러네요! 말씀해주신대로 operator 객체에 책임을 위임하니 훨씬 깔끔해졌습니다 👏🏻👏🏻👏🏻